google/cloud/speech/client.py

```diff
@@ -0,0 +1,157 @@
+                         :attr:`~.Encoding.FLAC`, :attr:`~.Encoding.MULAW`,
+                         :attr:`~.Encoding.AMR`, :attr:`~.Encoding.AMR_WB`
+
+        :type sample_rate: int
+        :param sample_rate: Sample rate in Hertz of the audio data sent in all
+                           RecognitionAudio messages. Valid values are: 8000-
+                           48000. 16000 is optimal. For best results, set the
+                           sampling rate of the audio source to 16000 Hz.
+                           If that's not possible, use the native sample rate
+                           of the audio source (instead of re-sampling).
+
+        :type language_code: str
+        :param language_code: (Optional) The language of the supplied audio as
+                             BCP-47 language tag. Example: "en-GB".
+                             If omitted, defaults to "en-US".
+
+        :type max_alternatives: int
+        :param max_alternatives: (Optional) Maximum number of recognition
+                                hypotheses to be returned. The server may
+                                return fewer than maxAlternatives.
+                                Valid values are 0-30. A value of 0 or 1
+                                will return a maximum of 1. Defaults to 1
+
+        :type profanity_filter: bool
+        :param profanity_filter: If True, the server will attempt to filter
+                                out profanities, replacing all but the
+                                initialcharacter in each filtered word with
+                                asterisks, e.g. "f***". If False or omitted,
+                                profanities won't be filtered out.
+
+        :type speech_context: list
+        :param speech_context: A list of strings (max 50) containing words and
+                              phrases "hints" so that the speech recognition
+                              is more likely to recognize them. This can be
+                              used to improve the accuracy for specific words
+                              and phrases. This can also be used to add new
+                              words to the vocabulary of the recognizer.
+
+        :rtype: list
+        :returns: A list of dictionaries. One dict for each alternative. Each
+                  dictionary typically contains two keys (though not
+                  all will be present in all cases)
+
+                  * ``transcript``: The detected text from the audio recording.
+                  * ``confidence``: The confidence in language detection, float
+                    between 0 and 1.
+        """
+
+        if content is None:
+            raise ValueError("content cannot be None")
+        if encoding is None:
+            raise ValueError('encoding cannot be None')
+        if sample_rate is None:
+            raise ValueError('sample_rate cannot be None')
```

Comment:
Please be consistent using single and double quotes. I prefer single quotes but @tseaver uses double quotes in these situations (i.e. situations with text intended to be read by a human)